easybuild/easyconfigs/f/futhark/futhark-0.19.5.eb

```diff
@@ -0,0 +1,26 @@
+easyblock = 'Tarball'
```

Comment:
@ysagon Thanks for this. Do you have any plans to work on a source, rather than binary build too?

Reply:
You welcome. I don't plan for now to work on the source installation.